proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -777,3 +947,22 @@ message SignedBuilderBidDeneb {
     bytes signature = 2 [(ethereum.eth.ext.ssz_size) = "96"];
 }
 
+message BuilderBidEpbs {
+    ExecutionPayloadHeaderEpbs header = 1;
+    ethereum.engine.v1.BlindedBlobsBundle blinded_blobs_bundle = 2;
+    bytes value = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+    bytes pubkey = 4 [(ethereum.eth.ext.ssz_size) = "48"];
+}
+
+message SignedBuilderBidEpbs {
+    BuilderBidEpbs message = 1 ;
+    bytes signature = 2 [(ethereum.eth.ext.ssz_size) = "96"];
+}
```

Comment:
We also don't need these anymore. It's represented as `SignedExecutionPayloadHeaderEnvelope`